libs/src/services/ethContracts/serviceProviderFactoryClient.js

```diff
@@ -197,12 +197,12 @@ class ServiceProviderFactoryClient extends GovernedContractClient {
       endpoint: info.endpoint,
       spID: parseInt(serviceId),
       type: serviceType,
-      blocknumber: info.blocknumber,
+      blockNumber: parseInt(info.blockNumber),
```

Comment:
this is going to cause breakage in creator node & dapp I think, can you double check?

Reply:
How can I check this? I do not expect it to break because blocknumber is not a field returned from info. A camelCased blockNumber is actually returned.

Reply:
Cool. No worries then maybe. Just run everything locally and upload a track. Or run mad dog :)